android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/MultipleChoicesAnswer.kt

```diff
@@ -13,32 +13,44 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import co.nimblehq.ic.kmm.suv.android.R
 import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+import co.nimblehq.ic.kmm.suv.domain.model.Answer
+import co.nimblehq.ic.kmm.suv.domain.model.AnswerInput
+import co.nimblehq.ic.kmm.suv.domain.model.Answerable
 
+object MultipleChoicesAnswerContentDescription {
+    private const val CHOICE = "CHOICE"
+
+    fun choice(index: Int) = "$CHOICE-$index"
+}
+
+@Suppress("NAME_SHADOWING")
 @SuppressLint("MutableCollectionMutableState")
 @Composable
 fun MultipleChoicesAnswer(
-    choices: List<String>,
-    onIndexesChange: (Set<Int>) -> Unit,
-    currentIndexes: Set<Int> = mutableSetOf()
+    answers: List<Answerable>,
+    onInputChange: (Set<AnswerInput>) -> Unit,
+    input: Set<AnswerInput> = mutableSetOf()
 ) {
-    var selectedIndexes by remember { mutableStateOf(currentIndexes) }
+    var input by remember { mutableStateOf(input) }
```

Comment:
same, please rename these param and var instead of putting `Suppress("NAME_SHADOWING")`

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/123/commits/07f6a3df329b5ed79084a3fe821669529cd47d33